tracer/build/_build/_build.csproj

```diff
@@ -58,13 +58,18 @@
     <Content Include="..\..\..\.azure-pipelines\ultimate-pipeline.yml" Link="ci\ultimate-pipeline.yml" />
     <Content Include="..\..\build_in_docker.sh" Link="boot\build_in_docker.sh" />
     <Content Include="..\..\build_in_docker.ps1" Link="boot\build_in_docker.ps1" />
+    <Content Include="..\..\..\.github\CODEOWNERS" Condition="Exists('..\..\..\.github\CODEOWNERS')" Link="CodeOwners\CODEOWNERS">
```

Comment:
What's with the `Condition`? 🤔 I'm guessing this is the problem that in some parts of the CI, we run from this folder and the file isn't "available"? But wouldn't that impact the links about too, like the azure-pipelines? 🤔

Reply:
Yes, some builds were failing because of that. I think that it fails because of CopyToOutputDirectory. The other ones do not have it.

Reply:
> I think that it fails because of CopyToOutputDirectory. 

That's kind of weird, but ok 😅